haskell-mode.el

```diff
@@ -956,7 +963,15 @@ successful, nil otherwise."
     (goto-char (point-min))
     (end-of-line)))
 
-
+(defun haskell-mode-set-interactive-prompt-state (&optional disabled)
```

Comment:
I though about it, but in the end I thought that having special function for this simple operation have at least two pros:
+ its name quite descriptive and self-explaining
+ it will ease further changes

What will you say?